cpp/src/bindings.cpp

```diff
@@ -66,6 +67,23 @@ PYBIND11_MODULE(_gridpy, m) {
 
     m.def("dump_network", &gridpy::dumpNetwork, "Dump network to a file in a given format");
 
+    py::class_<nominal_voltage_predicate>(m, "NominalVoltagePredicate")
```

Comment:
Maybe if in the Python API, we only expose vmin vmax we can avoid this object and just pass 2 floats?